packages/curation-market/clients/js/index.js

```diff
@@ -138,19 +429,60 @@ async function getBuyPriceFromCurve (supply, units, curveAddress) {
   return await curve.buyPrice(supply, units)
 }
 
+// replaces getSellPrice
+async function sellPriceFor(contributionId, amount) {
+  init()
+  const supply = await totalSupplyOf(contributionId)
+  return linearCurvePrice(supply.sub(amount), supply)
+}
+
+async function licenseSellPrice(licenseId) {
+  init()
+  const claim = await tokenClaimCheck.claims(licenseId)
+  const bytes32Id = await squadController.validLicenses(licenseId)
+  const contributionId = ethers.utils.parseBytes32String(bytes32Id)
+  return await sellPriceFor(contributionId, claim.amount)
+}
+
 async function getSellPrice (units, bondId) {
   init()
   const bondHash = ethers.utils.id(bondId)
   return await autoBond.getSellPrice(units, bondHash)
 }
 
+// replaces getMarketCap
+async function marketSize(contributionId) {
+  init()
+  const supply = totalSupplyOf(contributionId)
+  return linearCurvePrice(0, supply)
```

Comment:
In the future when there are multiple curves, this will need to look up the right curve to use